doc/source/whatsnew/v0.18.1.txt

```diff
@@ -321,6 +322,9 @@ Deprecations
 
 
 
+- Potential segfault in ``DataFrame.to_json`` when serialising ``datetime.time`` (:issue:`11473`).
```

Comment:
`DatetimeTZBlock` was covered by #10778 mentioned [here](https://github.com/pydata/pandas/pull/12802/files#diff-95743d156b5649e3cce3e70750d36248R397). (#11473 is a separate issue related to  `datetime` objects)